examples/stm32h7/src/bin/signal.rs

```diff
@@ -0,0 +1,41 @@
+#![no_std]
+#![no_main]
+#![feature(type_alias_impl_trait)]
+
+use defmt::{info, unwrap};
+use embassy::executor::Spawner;
+use embassy::time::{Duration, Timer};
+use embassy_stm32::Peripherals;
+
+use defmt_rtt as _;
+use embassy::channel::Signal;
+// global logger
```

Comment:
this comment should go next to `use defmt_rtt`

Reply:
done, I rearranged the imports a little to make things a bit clearer and then reformatted with rustfmt.